### PR TITLE
Improvements for multinic test with NetworkManager

### PR DESCRIPTION
**Pull request by bengal**
This pull request hopefully fixes the 50-multinic test that used to randomly fail with the network-manager module.

## Checklist
- [X] I have tested it locally
- [X] I have reviewed and updated any documentation if relevant
- [X] I am providing new code and test(s) for it

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -72,7 +72,7 @@ jobs:
                         "30",
                         "35",
                         "40",
-                        # "50", # times out
+                        "50",
```
Comment (aafeijoo-suse) [inline .github/workflows/integration.yml:75]: But I'm not sure if we should re-enable test 50, although it's working now with the NetworkManager, it still fails with the network-legacy module:

https://github.com/dracutdevs/dracut/actions/runs/4076254978/jobs/7075949251

File: test/TEST-50-MULTINIC/test.sh [hunk 2]
```diff
@@ -365,7 +362,7 @@ test_setup() {
     # Make server's dracut image
     "$basedir"/dracut.sh -l -i "$TESTDIR"/overlay / \
         -m "dash rootfs-block debug kernel-modules watchdog qemu network network-legacy" \
-        -d "af_packet piix ide-gd_mod ata_piix ext3 sd_mod nfsv2 nfsv3 nfsv4 nfs_acl nfs_layout_nfsv41_files nfsd e1000 i6300esb ib700wdt" \
```
Comment (aafeijoo-suse) [inline test/TEST-50-MULTINIC/test.sh:368]: Multiple `modprobe: FATAL: Module ib700wdt not found in directory /lib/modules/6.1.9-200.fc37.x86_64`:

https://github.com/dracutdevs/dracut/actions/runs/4076254978/jobs/7075949251#step:4:386

`ib700wdt` module required by: https://github.com/dracutdevs/dracut/blob/master/modules.d/04watchdog/watchdog.sh

**Review by aafeijoo-suse — COMMENTED** [review]
Thanks for you help here!

File: test/TEST-50-MULTINIC/test.sh (hunk 2 above)
Comment (aafeijoo-suse) [reply test/TEST-50-MULTINIC/test.sh:368]: > Not sure if this is in scope for this PR, but we could consider removing ib700wdt from watchdog module.

Indeed, out of the scope of this PR.

Comment (bengal) [reply test/TEST-50-MULTINIC/test.sh]: Okay, I'll file a separate MR for the watchdog changes.

File: .github/workflows/integration.yml (hunk 1 above)
Comment (bengal) [reply .github/workflows/integration.yml]: Okay

**Comment by bengal** [comment]
I have removed the watchdog changes and left the test disabled for now since there are still some failures. Nevertheless, I think these changes should be merged because they improve things.

**Review by aafeijoo-suse — APPROVED** [review]

**Pull request merged**